doc/whats-new.rst

```diff
@@ -44,6 +44,13 @@ New Features
   option for dropping either labels or variables, but using the more specific methods is encouraged.
   (:pull:`3475`)
   By `Maximilian Roos <https://github.com/max-sixty>`_
+- :py:meth:`Dataset.map` & :py:meth:`GroupBy.map` & :py:meth:`Resample.map` have been added for 
```

Comment:
I think we need these in api.rst? or is it automatically added?

Reply:
Ah, thanks @dcherian , good spot